### ISSUE TITLE
[user pindecompvab1989] Report user for malicious or infringing images

### ISSUE THREAD

**Issue by afirth** [docker/hub-feedback#2232]
https://hub.docker.com/u/pindecompvab1989

User has ~1000 repos purporting to offer free copies of commercial software, among other things

**Comment by SlevinWasAlreadyTaken** [comment]
I agree, it also seems the images are malicious and do not what they are declared to be (illegal in all cases).

**Comment by vdamery** [comment]
This user has been deactivated.

Closing the issue.
